include/Random.h

```diff
@@ -32,6 +33,13 @@ class RandomNumberGenerator {
     void Generate(mpfr_ptr);
 
    private:
+    std::mt19937_64 small_rng_;
+    static constexpr double neg_frac_ = 1.0/3.0;
```

Comment:
The naming convention for compile-time constants (from Google's style guide) is `kMyConstant`